scripts/linux/publishReleasePackages.sh

```diff
@@ -119,10 +121,10 @@ process_args() {
             "-d" | "--dir") save_next_arg=2 ;;
             "-w" | "--wdir") save_next_arg=3 ;;
             "-s" | "--server") save_next_arg=4 ;;
-            "-g" | "--ghubpat") save_next_arg=5 ;;
-            "-v" | "--version") save_next_arg=6 ;;
+            "-r" | "--repo-name") save_next_arg=5 ;;
+            "-g" | "--ghubpat") save_next_arg=6 ;;
```

Comment:
nit: can do we `gh-pat` instead please?

Reply:
Done. Actually, I'm tempted to just remove that parameter since we aren't using it anyway. We're just declaring GITHUB_PAT in the environment, which is a better way to pass secrets. Keeping it for now though...